frontend/src/reducers/user.js

```diff
@@ -0,0 +1,31 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const user = createSlice({
+    name: 'user',
+    initialState: {
+        username: null,
+        accessToken: null,
+        errors: null
+    },
+    // actions 
+    reducers: {
+        setUsername: (store, action) => {
+            store.username = action.payload;
+        },
+        setAccessToken: (store, action) => {
+            store.accessToken = action.payload;
+        },
+        setErrors: (store, action) => {
+            store.errors = action.payload;
+        },
+        logout: (state, action) => {
+            console.log("Logging out")
+            state.accessToken = null;
+            state.user = {}
+        },
+    }
+
+});
+
+export default user;
```

Comment:
You need to store your user in the local storage to prevent logging out on refresh.

Reply:
Hi Karol, we have now added local storage for signing in  :)